src/extension/edge/backgroundScriptsAPIBridge.js

```diff
@@ -0,0 +1,1044 @@
+if (!Range.prototype["intersectsNode"]) {
```

Comment:
did you write these javascript files? If not, we should have some sort of link to where you found them/attribution to who wrote them